bundle/manifests/kueue-operator.clusterserviceversion.yaml

```diff
@@ -296,9 +296,9 @@ spec:
     name: Red Hat, Inc
     url: https://github.com/openshift/kueue-operator
   relatedImages:
-    - image: registry.redhat.io/kueue-tech-preview/kueue-rhel9@sha256:9a0b85b0752135694193befe00d10f7abdc0c99c6b9f295d59b237fa0fe3e6d2
+    - image: registry.redhat.io/kueue/kueue-rhel9@sha256:73390c4d95bacb5b4d96eac7f82e76fbba4161beb1220e5a4924fdef3e3a8e01
       name: operand-image
     - name: must-gather
-      image: registry.redhat.io/kueue-tech-preview/kueue-must-gather-rhel9@sha256:e5306c95bf6c5f55be6e791fcb66fe9b34f4f982536f4f90f8ed74acf3f6c31d
+      image: registry.redhat.io/kueue/kueue-must-gather-rhel9@sha256:e5306c95bf6c5f55be6e791fcb66fe9b34f4f982536f4f90f8ed74acf3f6c31d
```

Comment:
which image digest is this one?
should we be using the one listed here https://catalog.redhat.com/software/containers/kueue-tech-preview/kueue-must-gather-rhel9/682643f5fff6dd793909757c?container-tabs=gti&gti-tabs=registry-tokens?

Reply:
Once we have this merged, I can trigger a prod release and that will publish these images to GA. And then they would exist in kueue/kueue-must-gather-rhel9.

But right now they don't exist there.

In our CI we actually remap to the quay repos. But for production we need to reference these images.

its stupid..